Tools/WinMLRunner/src/CommandLineArgs.h

```diff
@@ -98,19 +98,18 @@ class CommandLineArgs
     void ToggleTerseOutput(bool terseOutput) { m_terseOutput = terseOutput; }
 
     void SetModelPath(const std::wstring& modelPath) { m_modelPath = modelPath; }
-    void SetTensorOutputPath(const std::wstring& tensorOutputPath) { m_tensorOutputPath = tensorOutputPath; }
+    void SetPerIterationDataPath(const std::wstring& perIterationDataPath)
+    {
+        m_perIterationDataPath = perIterationDataPath;
+    }
     void SetInputDataPath(const std::wstring& inputDataPath) { m_inputData = inputDataPath; }
     void SetNumThreads(unsigned numThreads) { m_numThreads = numThreads; }
     void SetThreadInterval(unsigned threadInterval) { m_threadInterval = threadInterval; }
     void SetTopK(unsigned k) { m_topK = k; }
-    void SetPerformanceCSVPath(const std::wstring& performanceCSVPath)
-    {
-        m_perfOutputPath = performanceCSVPath;
-        m_perfOutput = true;
```

Comment:
Since ```m_perfOutput = true``` is getting removed, can you add a small  "TogglePerfOutput(bool perfOutput)" method?